### PR TITLE
Update README with connection to db via psql

### PR DESCRIPTION
**Pull request by eleijonmarck**
For newbies <-- it would be great to have a clear example of how to connect to the database and see the schema.
This is essential I would say to later than see how the impact of a change in `current.sql` will take immediate effect (i.e. migration happening in realtime).

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ and may make it harder to reproduce issues.
 run
 [docker-volume-watcher](https://github.com/merofeev/docker-windows-volume-watcher)
 
+#### DB tool:
+
+To connect to the newly created container db with psql/{db tool} you connect to it by default if not anything else specified via port 6543 on localhost with the `DATABASE_NAME` specificed as such:
+
+```bash
+$ psql postgres://$DATABASE_NAME@localhost:6543
```
Comment (benjie) [inline docker/README.md]: The database name goes into the path of the connection string, and you may need to add authentication credentials. These settings are from `.env`:

```suggestion
$ psql "postgres://$DATABASE_OWNER:$DATABASE_OWNER_PASSWORD@localhost:6543/$DATABASE_NAME"
```

File: docker/README.md [hunk 2]
```diff
@@ -46,6 +46,14 @@ and may make it harder to reproduce issues.
 run
 [docker-volume-watcher](https://github.com/merofeev/docker-windows-volume-watcher)
 
+#### DB tool:
+
+To connect to the newly created container db with psql/{db tool} you connect to it by default if not anything else specified via port 6543 on localhost with the `DATABASE_NAME` specificed as such:
```
Comment (benjie) [inline docker/README.md]:
```suggestion
To connect to the container database with psql or another database tool, use port 6543 on localhost and populate the `DATABASE_NAME`, `DATABASE_OWNER` and `DATABASE_OWNER_PASSWORD` from your `.env` file:
```

File: docker/README.md (hunk 1 above)
Comment (eleijonmarck) [reply docker/README.md]: Cool! Do we need to say something about .env you think?

**Comment by eleijonmarck** [comment]
@benjie did something happen between the fixes?

File: docker/README.md (hunk 1 above)
Comment (benjie) [reply docker/README.md]: Not sure

**Comment by benjie** [comment]
It's a lint error, you need to run `yarn lint:fix` and commit the result :+1:

**Pull request merged**